### ISSUE TITLE
Using a batch from the training set to represent the test/validation set

### ISSUE THREAD

**Issue by psteinb** [VLL-HD/conditional_invertible_neural_networks#8]
just saw that
https://github.com/VLL-HD/conditional_invertible_neural_networks/blob/45dc7250ebfecf10d1a278edafde0fe899f30aa1/mnist_minimal_example/train.py#L35
uses a given batch of training data to calculate the loss for the validation? Was this done on purpose? The `data` module offers a `testloader` as far as I could see.
